python/ray/tests/test_metrics_agent.py

```diff
@@ -587,61 +587,64 @@ def test_operation_stats(monkeypatch, shutdown_only):
         "ray_operation_queue_time_ms_bucket",
         "ray_operation_active_count",
     ]
-    addr = ray.init()
-    remote_signal = SignalActor.remote()
 
-    @ray.remote
-    class Actor:
-        def __init__(self, signal):
-            self.signal = signal
+    with monkeypatch.context() as m:
```

Comment:
No need to have a context in this case, you can directly to `monkeypatch.setenv()`